repos/system_upgrade/common/actors/kpatch/actor.py

```diff
@@ -0,0 +1,26 @@
+from leapp.actors import Actor
+from leapp.libraries.stdlib import api
+from leapp.libraries.common.rpms import has_package
+from leapp.models import CopyFile, TargetUserSpacePreupgradeTasks, InstalledRedHatSignedRPM
+from leapp.tags import FactsPhaseTag, IPUWorkflowTag
+
+PLUGIN_PKGNAME = "kpatch-dnf"
+CONFIG_PATH = "/etc/dnf/plugins/kpatch.conf"
+
+class Kpatch(Actor):
```

Comment:
please rename the actor (including dirname) to e.g. `CheckKpatch` (dir `checkkpatch`)  to make it more clear it's basically the check actor.